libs/java/server_common/src/test/java/com/yahoo/athenz/common/server/store/MockCloudStore.java

```diff
@@ -0,0 +1,104 @@
+/*
+ *  Copyright 2020 Verizon Media
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I had to copy this file and the other "MockChangeLog" classes from ZTS as they are shared between tests in ZTS and in Common (and test classes are not included in the common jar). 
There are 2 ways to overcome this code duplication:
1. Move these classes outside of test directory
2. Add a "packaging" step for tests in common that will create a jar that includes tests. We can then make ZTS and ZMS dependent on that jar (see https://dzone.com/articles/sharing-test-classes-between-multiple-modules-in-a).
For now, I thought it's best to leave it as is. We can fix it in a later PR if needed.